src/main/java/com/lorepo/icplayer/client/module/text/TextParser.java

```diff
@@ -290,7 +291,11 @@ private String matchGap(String expression, Map<String,String> gapOptions) {
 	private DomElementManipulator createGapInputElement(String id, String answer, Map<String,String> gapOptions) {
 		DomElementManipulator inputElement = new DomElementManipulator("input");
 		inputElement.setHTMLAttribute("id", id);
-		inputElement.setHTMLAttribute("type", "edit");
+		if (!isNumericOnly) {
```

Comment:
Zastanawiam się na ile to jest teraz potrzebne, skoro ta informacja jest w GapInfo i GapWidget już ją odpowiednio wykorzystuje. Powinno być jedno miejsce, w którym ten atrybut jest ustawiany.